system/views/admin/datamanager/_objectDataTable.cfm

```diff
@@ -171,7 +171,15 @@
 						</th>
 					</cfif>
 					<cfloop array="#args.gridFields#" index="fieldName">
-						<th data-field="#ListLast( fieldName, '.' )#">#translatePropertyName( args.objectName, fieldName )#</th>
+						<th data-field="#ListLast( fieldName, '.' )#">
+							#translatePropertyName( args.objectName, fieldName )#
```

Comment:
Should this have a context of 'listing', i.e.

```translatePropertyName( args.objectName, fieldName, "listing" )```

Reply:
thanks for notice this, my bad was missed out when copying over.